/forms v2

ondrae opened the pull request:
This PR adds a page of compliance considerations that Federalists customer must think through when adding a form to their website.

[![CircleCI](https://circleci.com/gh/18F/federalist.18f.gov/tree/add-forms-docs.svg?style=svg)](https://circleci.com/gh/18F/federalist.18f.gov/tree/add-forms-docs)

[:sunglasses: PREVIEW](https://cg-06ab120d-836f-49a2-bc22-9dfb1585c3c6.app.cloud.gov/preview/18f/federalist.18f.gov/add-forms-docs/documentation/forms/)

davemcorwin commented on pages/documentation/forms.md:
```diff
@@ -0,0 +1,87 @@
+    if (/(.gov|.mil)$/.test(email.value)) {
+      // redirect that works on preview builds too
+      const current_url = window.location.href;
+      const new_url = current_url.replace("forms", "form-services")
```
This still doesn't work in the preview because the URL includes "forms" in multiple places :)
```suggestion
      // Just replace the url wit the entire one we want
      const new_url = "{{ site.baseurl }}/documentation/form-services";
```

ondrae replied:
Yeah, I threw my hands up and went about the rest of my day, haha. Thanks for this.

ondrae commented:
Okay @amirbey and @davemcorwin this PR is ready to be reviewed, squashed, and merged.

It adds a single page of compliance advice for Federalist customer who want to add a form to their website. We dropped all the form service recommendations and advice, due to legal pressure.

ondrae commented:
We tried a few different ways to get around giving vendor specific advice, like an email gate that required a federal email. At the same time we were trying these hacks, our content on the page kept getting reduced and more vague, until it didn't feel worth it any more. So we pulled it.

davemcorwin commented on pages/documentation/forms.md:
```diff
@@ -0,0 +1,54 @@
+
+
+## A note about form services
+Federalist pages are static websites. This means there is no backend for you to log into in order to get your agency’s form data. You’ll need to use a separate form service to use a digital form on your agency’s site. You should check with your agency's Office of Chief Information Officer for what services are licensed and approved for your use.
```
```suggestion
Federalist pages are static websites. This means that while you can add a form to your site, there is no mechanism to handle the form submission to store and see the results. You’ll need to use a separate form service to use a digital form on your agency’s site. You should check with your agency's Office of Chief Information Officer for what services are licensed and approved for your use.
```

anjunainaustin commented:
Thanks @davemcorwin ! I just added one last minute change to PA stuff that OMB noticed, but it looks good to go now, and I think your edit improved this too. Once this is up on the live site I need to tell MVB blog folks bc they are waiting on that in order to publsh the blog. Can we get this live this week?
